packages/styles/scss/patterns/blocks/contentgroupcards/_contentgroupcards.scss

```diff
@@ -66,6 +62,11 @@
       }
     }
 
+    .#{$prefix}--card-link__title,
+    .#{$prefix}--card-link__content {
+      width: 90%;
+    }
```

Comment:
This should be at the card level I would imagine. I think we want this 90% across all cards. @oliviaflory correct me if I'm wrong.